pkg/build/controller/build/build_controller.go

```diff
@@ -834,4 +837,27 @@ func setBuildCompletionTimestampAndDuration(build *buildapi.Build, podStartTime
 		update.setStartTime(*startTime)
 	}
 	update.setDuration(now.Rfc3339Copy().Time.Sub(startTime.Rfc3339Copy().Time))
+
+	if pod != nil && len(pod.Status.ContainerStatuses) != 0 && pod.Status.ContainerStatuses[0].State.Terminated != nil {
+		msg := pod.Status.ContainerStatuses[0].State.Terminated.Message
+		if len(msg) != 0 {
+			parts := strings.Split(msg, "\n")
```

Comment:
Since this is going to end up in the web console, we need it more cleaned up than what is already being done.  Ensuring we have full lines (no starting in the middle of a line) is important in terms of confusing users, i think.